packages/backend/src/services/specimenService/resources/specimen/data/migrations/addIndex.js

```diff
@@ -0,0 +1,37 @@
+const relationshipIndexes = [
+  {
+    name: 'idx_spec_rel_phys',
```

Comment:
not a fan of arbitrary names: phys, agent, place, taxa are 4 or 5 chars, singular, plural, and abbreviation all mixed.

how about something like this:
```javascript
const relationshipKeys = [
  'normalizedAgents',
  'physicalObjects',
  'places',
  'taxa',
]

const relationshipIndexes = relationshipKeys.map(relationship => {
  return {
    name: `idx_specimen_relationship_${relationship}`,
    relationship,
  }
})
```

Reply:
#167cf3f